docs/admin/projects.rst

```diff
@@ -638,16 +638,16 @@ Committer name
 Name of the committer used for Weblate commits, the author will always be the
 real translator. On some VCSs this might be not supported.
 
-Default value can be changed by :setting:`DEFAULT_COMMITER_NAME`.
+The default value can be changed by :setting:`DEFAULT_COMMITER_NAME`.
 
 .. _component-committer_email:
 
 Committer e-mail
 ++++++++++++++++
 
-Email of committer used for Weblate commits, the author will always be the
-real translator. On some VCSs this might be not supported. The default value
-can be changed in :setting:`DEFAULT_COMMITER_EMAIL`.
+E-mail address of committer used for Weblate commits. The author will always
```

Comment:
```suggestion
E-mail address of the committer used for Weblate commits. The author will always
```

Reply:
Is "E-mail address for commits sent from Weblate" enough?

Reply:
I'd prefer to keep words `committer` and `author`, as those are the settings name used in `git config`... It makes more clear what is meant as those are opposed to each other in the snippet...

Reply:
Yeah, it just isn't a very sleek sentence, but that is another problem.